use std::error::Error;

pub fn solve(input: Vec<String>) -> Result<(String, String), Box<dyn Error>> {
    Ok(("".to_string(), "".to_string()))
}
